ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-505] Display frozen income savings assets in capital details

## Description of change
Displays the step "Does your client have any income, savings or assets under a restraint or freezing order?" in the capital assessment section, if it was not asked in the income assessment section

## Link to relevant ticket
[CRIMAPP-505](https://dsdmoj.atlassian.net/browse/CRIMAPP-505)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

<img width="1173" alt="Screenshot 2024-03-18 at 15 54 49" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/6f56aae5-b436-4160-afd7-fc078d2d403d">
<img width="759" alt="Screenshot 2024-03-18 at 12 19 42" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/574e92cf-6d0a-4119-9218-29f8d30646c0">
<img width="759" alt="Screenshot 2024-03-18 at 12 19 28" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/afbfc651-163d-4bb2-b51f-2c3888e5a657">
<img width="781" alt="Screenshot 2024-03-18 at 15 55 50" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/70fc63bd-f546-4ee2-b701-8663d6411f00">

With added hint text
<img width="781" alt="Screenshot 2024-03-18 at 16 09 45" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/0bf8adfc-8eb9-4c32-8aca-7e53706ddac4">

## How to manually test the feature
1. Start a application, get through the application till you get to the income section 
2. When going through the income section select **yes** on the `/steps/income/clients_income_before_tax` screen. This will ensure the income, savings assets question is displayed in the capital section 
3. Then go through the application until you reach the capital assessment questions and specifically the trust fund question 
4. After this question, you should see the income savings assets question 
5. Try to save and continue without adding an answer - you should receive an error message
6. Select no and then go through the rest of the application and test you can see the answer in the review application screen after  the trust fund section 
7. Click the change link and test that it takes you back to the right screen `steps/capital/income_savings_assets_under_restraint_freezing_order`
8. Then change your answer to yes and click save and come back later. And confirm on the review application screen that the answer has been changed to yes 
9. Submit the application and check it is displayed on the completed application page 
10. Then start a new application and go through to the income screen and select **no** on the `/steps/income/clients_income_before_tax` screen. This will ensure the income, savings assets question is displayed in the income section and **not** displayed in the capital section

[CRIMAPP-505]: https://dsdmoj.atlassian.net/browse/CRIMAPP-505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
